### PR TITLE
Upon login notify user that the username attempted to login with does not exist

### PR DESCRIPTION
**Pull request by xtrinch**
Notify user that user with the username attempted to login with does not exist.

I think it would be great if there would be an option to notify the user that he is trying to login with an account that does not exist.

### CODE REVIEW

File: djoser/serializers.py [hunk 1]
```diff
@@ -69,6 +70,8 @@ def validate(self, attrs):
                 raise serializers.ValidationError(self.error_messages['inactive_account'])
             return attrs
         else:
+            if User.objects.filter(**{User.USERNAME_FIELD: attrs.get(User.USERNAME_FIELD)}).first() is None:
```
Comment (piotr-szpetkowski) [inline djoser/serializers.py]: A better condition to check for would be:
`not User.objects.filter(**{User.USERNAME_FIELD: attrs.get(User.USERNAME_FIELD)}).exists()`

**Comment by piotr-szpetkowski** [comment]
Also, we should probably pull it under a setting feature flag.

**Comment by jasonrfarkas** [comment]
I'm not a security expert. (I was just was looking for examples of how others handled drj api for users.) But, I believe the suggested feature is actually a security concern. In general when a user doesn't have permission to view a resource the best practice is to just tell them it does not exist. You don't want to let them know that things exist but they can't access them, even more so with users.

**Comment by piotr-szpetkowski** [comment]
Actually it seems like an author does not express an interest in further development of this feature. Therefore I'm going to close it now, however if it turns out that someone needs it than he/she is welcome to re-open it under new PR.

**Pull request closed without merging**